velox/exec/tests/WindowTest.cpp

```diff
@@ -132,7 +132,7 @@ TEST_F(WindowTest, rowBasedStreamingWindowOOM) {
 
       VELOX_ASSERT_THROW(
           readCursor(params, [](Task*) {}),
-          "Exceeded memory pool capacity after attempt to grow capacity through arbitration.");
+          "Exceeded memory pool cap of");
```

Comment:
@ccat3z Why need this change?

Reply:
https://github.com/facebookincubator/velox/pull/10506 changed exception message, which not included in branch 1.2